keyboards/levinson/readme.md

```diff
@@ -3,18 +3,24 @@ Levinson
 
 A split 40% split 4x12 ortholinear keyboard made and sold by Keebio. It's essentially a Let's Split with LED backlight support and 2u thumb key support. [More info at Keebio](https://keeb.io).
 
-Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)  
-Hardware Supported: Pro Micro  
-Hardware Availability: [Keebio](https://keeb.io)  
+Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)
+Hardware Supported: Pro Micro, Elite-C
+Hardware Availability: [Keebio](https://keeb.io)
 
 Make example for this keyboard (after setting up your build environment):
 
     make levinson/rev2:default
 
-Example of flashing this keyboard:
+Example of flashing this keyboard with Pro micro:
 
     make levinson/rev2:default:avrdude
 
+Example of flashing this keyboard with Elite-C:
+
+    Add `BOOTLOADER = atmel-dfu` to the `rules.mk` file. Hold the reset button on the Elite-C board, or the reset button on the keyboard if already assembled, then release the button and run:
```

Comment:
Well then, maybe this would be better: 
```md
Note: You'll need to add `BOOTLOADER = atmel-dfu` to your `rules.mk` before you compile it, to ensure that resetting the board works properly. 
```

And actually, yes, this has everything  to do with resetting the board.  How QMK resets the board is very much  dependant on the type of bootloader defined and used.  If you have the wrong bootloader, it may not work properly and never actually enter the bootloader. 

This is explicitly why I always ask about the bootloader for new keyboards, actually. 

https://github.com/qmk/qmk_firmware/blob/ee700b2e831067bdb7584425569b61bc6329247b/tmk_core/common/avr/bootloader.c#L99-L171